docs/coding-standards/javascript-react.mdx

```diff
@@ -0,0 +1,297 @@
+    "@aoeu/eslint-config/testing"
+  ]
+}
+```
+
+The above configuration includes [Jest](https://jestjs.io/) and [TestCafe](https://testcafe.io/) globals, and rules specific to those tools and common issues relevant to test fixtures.
+
+### Usage
+
+Once you've added the configuration to your project, you can/should add the following task/script to your projects `package.json` file to execute linting:
+
+```json
+// package.json
+{
+  "name": "your-project",
+  "scripts": {
+    "eslint": "eslint src testing next.config.js --ext js,jsx"
+  }
+}
+```
+
+In the above task, `eslint` is the command to run, and `src`, `testing` and `next.config.js` are the files and folders that you wish for ESLint to be run in, and the `--ext` argument tells ESLint the file extensions to look for to perform linting in.
+
+This task/script can be run from your terminal/cli with the following command:
+
+```
+yarn run eslint
+```
+
+And you will/should see the output of the command in your terminal. In our projects, we expect all pull requests (PR) to be submitted with no ESLint errors or warnings, and this should be configured as part of your projects continuous integration workflow run as a GitHub action, to be run each time a change is commited to a PR.
+
+### Editor Integration
+
+Most modern code editors can have ESLint integrated into them such that it is run whenever a change is saved to a file, so you should be warned of ESLint errors and warnings in nearly real-time during the course of your development.
+
+The following extensions are available for these known editors.
+
+* Visual Studio Code: [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+* Sublime Text: [SublimeLinter](https://packagecontrol.io/packages/SublimeLinter) with the [ESLint plugin](https://packagecontrol.io/packages/ESLint) and the [SublimeLinter-eslint interface](https://packagecontrol.io/packages/SublimeLinter-eslint).
+* Vim: [vim-eslint plugin](https://github.com/dgraham/vim-eslint)
+* JetBrains WebStorm: [ESLint plugin](https://plugins.jetbrains.com/plugin/7494-eslint)
+
+Integrations for other editors are available, but those are the most popular ones in use at this time.
+
+## JavaScript, the important parts 😏
+
+While we like Google's JavaScript style guide as a general baseline, we have defined some of our own customizations to that guide that support our organizational preferences. This section will attempt to outline and highlight the significant distinctions between Google's ESLint config, and our customizations to it, that are the most signficant for you to be aware of.
+
+### [`semi`](https://eslint.org/docs/rules/semi#require-or-disallow-semicolons-instead-of-asi-semi)
+
+We exclude the use of semicolons at the end of statements in our JavaScript.
+
+ESLint rule definition:
+```
+"semi": ["error", "never"]
+```
+
+#### **Examples**
+
+Bad:
+
+```
+const foo = "bar";
+```
+
+Good:
+
+```
+const foo = bar
```

Comment:
Does 'bar' need the "quotes" around it? It looks like _Good_ = without the semi-colon and the quotes.

Reply:
I think in this case `bar` is another variable name so wouldn't need the quotes.

Reply:
✅